articles/reference/dashboard/settings-api.md

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
Are we sure we want this to be /reference/dashboard vs /dashboard/reference?

Reply:
Changed to /dashboard/reference and updated referring links.